### PR TITLE
Add boot message, disable installer on boot

### PR DESCRIPTION
**Pull request by evil-brain**
Add a message printed on boot that gives basic information and
instructions to login as root and run clr-installer to install onto this
sytem.

Signed-off-by: Brian J Lovin <brian.j.lovin@intel.com>

Changes proposed in this pull request:
- Remove autolaunch of clr-installer on boot
- Add pre-login message with Clear information and instructions to install

### CODE REVIEW

**Comment by evil-brain** [comment]
Got feedback, need to remove the service from the other yaml files, and remove it's installation from the Makefile.

**Comment by evil-brain** [comment]
Quite a few changes, updated the commit msg with them, but:
 * Dropped all configs related to clr-installer.service and the file itself (installer and live img)
 * Changed PS1 after logging in to show only "root@clr-live ${pwd}" (**only on installer**)

File: scripts/add-login-issue.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+echo "Creating custom issue file for $1"
+
+echo "Welcome to the Clear Linux live image!
```
Comment (mdhorn) [inline scripts/add-login-issue.sh]: Please change to "Welcome to the Clear Linux* OS live image!"

File: scripts/add-login-issue.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+ * Documentation:     https://clearlinux.org/documentation/clear-linux
```
Comment (mdhorn) [inline scripts/add-login-issue.sh]: Please drop the "clear-linux" from the URL, it should redirect.

File: scripts/add-login-issue.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ * Community Support: https://clearlinux.org/community
+
+To install Clear Linux onto this sytem please login as root,
+enter a new temporary password, and run clr-installer.
```
Comment (mdhorn) [inline scripts/add-login-issue.sh]: Could we change from:
`enter a new temporary password, and run clr-installer.`
to:
`enter a new temporary password, and run:\n
clr-installer`

File: scripts/live-desktop-post-install.sh [hunk 4]
```diff
@@ -17,6 +17,9 @@ ${HOOKDIR}/local-telemetry-post.sh ${CHROOTPATH}
 # Delay booting to give user a change to change boot params
 ${HOOKDIR}/wait-to-boot-post.sh ${CHROOTPATH}
 
+# Add issue (pre-login message) to inform user of how to run the installer
+scripts/add-login-issue.sh ${CHROOTPATH}
```
Comment (mdhorn) [inline scripts/live-desktop-post-install.sh]: Please drop from here as we already call it indirectly.

**Comment by mdhorn** [comment]
@evil-brain If you rebase you'll pick up Reagan's makefile updates and the Travis should pass for you.

**Comment by mdhorn** [comment]
The travis was the only thing not passing due to timeout.
I've push a new Makefile to fix this. 
I need this change so I'm merging.

**Pull request merged**